helm-swoop.el

```diff
@@ -26,7 +26,7 @@
 ;;; Commentary:
 
 ;; List the all lines to another buffer, which is able to squeeze
-;; by any words you input. At the same time, the original buffer's
+;; by any words you input.  At the same time, the original buffer's
```

Comment:
What's the matter with this extra space?

Reply:
Please turn on flycheck in helm-swoop.el.
You can see below warnings.
```
 helm-swo…    29     warning         There should be two spaces after a period (emacs-lisp-checkdoc)
 helm-swo…    60     warning         There should be two spaces after a period (emacs-lisp-checkdoc)
 helm-swo…    89     warning         There should be two spaces after a period (emacs-lisp-checkdoc)
```

Reply:
Ok, refer to [this StackExchange answer](https://english.stackexchange.com/questions/192070/how-widely-used-is-the-two-space-sentence-ending-convention). Use double-space is acceptable because monospaced-font is widely used in programming.